src/store/participantsStore.js

```diff
@@ -85,16 +85,14 @@ const getters = {
 	 * Gets the array of external session ids.
 	 *
 	 * @param {object} state - the state object.
-	 * @param {object} rootState - the root state object.
 	 * @return {Array} the typing session IDs array.
 	 */
-	externalTypingSignals: (state, rootState) => (token) => {
-		if (state.typing[token]) {
-			return Object.values(state.typing[token]).filter(id => {
-				return rootState.getSessionId() !== id
-			})
+	externalTypingSignals: (state) => (token) => {
+		if (!state.typing[token]) {
+			return []
 		}
-		return []
+
+		return Object.entries(state.typing[token])
 	},
```

Comment:
I'm not sure how we should divide external and internal typings (we do it in the component as well).

but the issue in the previous solution is that `rootState` is the 3rd getter's params, not 2nd.